File: AreaTriangulo.dart
import 'dart:io'; 
void main() { 
  //Definir Variables y otros 
  print('Ejercicio 01: Area Triangulo'); 
  //Datos de Entrada 
  print("Ingrese Base: "); 
  var b=int.parse(stdin.readLineSync()); 
  print("Ingrese Altura: "); var h=int.parse(stdin.readLineSync()); 
  //Proceso 
  var area=(b*h)/2;
  //Datos de Salida 
  print("El Area de Triangulo es: $area"); 
  }

File: AreaTriangulo.java
import java.util.Scanner;
class AreaTriangulo{
  static Scanner teclado=new Scanner (System.in);
  public static void main(String[]arg){
    //definición de variables
    System.out.println("Ejercicio 01: Hallar Área del Triangulo");
    int b,h,area;
    //datos de entrada
    System.out.println("Ingrese Base: ");
    b=teclado.nextInt();
    System.out.println("Ingrese Base: ");
    h=teclado.nextInt();
    //Proceso
    area=b*h/2;
    System.out.print("El área del trinagulo es: " + area);

  }
}